src/common/components/SolBalance.tsx

```diff
@@ -14,7 +14,7 @@ export const SolBalance = () => {
   } else if (data === undefined || data === null) {
     amountToShow = ' ';
   } else if (data > 0) {
-    amountToShow = `${(data / LAMPORTS_PER_SOL).toFixed(2)} SOL`;
+    amountToShow = `${(data / LAMPORTS_PER_SOL).toFixed(2)}`;
```

Comment:
Let's leave this sort of change for another Pull Request. Balance can be in SOL for now.

Reply:
Deal, was rushing to prod ha